packages/dx-grid-core/src/plugins/table-column-resizing/computeds.js

```diff
@@ -3,12 +3,17 @@ const UNSET_COLUMN_WIDTH_ERROR = [
   'The TableColumnResizing plugin requires that all columns have the specified width.',
 ].join('\n');
 
+const findCurrentColumn = (array, columnName) =>
+  array.find(elem => elem.columnName === columnName);
```

Comment:
@gsobolev what will you say about this extracted function?

Reply:
`findCollumn` or `findColumnByName` ?